client/src/components/CreatePost/CreatePost.js

```diff
@@ -133,8 +134,12 @@ const Step4 = () => {
         <TitleStep fontSize={typography.size.xlarge} currentStep={currentStep}>
           Success
         </TitleStep>
-        <CreateProfileButton primary>Create Profile</CreateProfileButton>
-        <CreateOrgLink to={""}>Skip</CreateOrgLink>
+        {/* <CreateProfileButton primary>Your Post have been posted successfully</CreateProfileButton>*/}
+        <CreatePostSuccessMessagePopUpDiv>
+          Posted Successfully
+        </CreatePostSuccessMessagePopUpDiv>
+        {/* <CreateOrgLink onClick={() => setCurrentStep(1)}>Please click to continue</CreateOrgLink> */}
+        <CreateOrgLink to={"/feed/:id"}>Please click to continue</CreateOrgLink>
```

Comment:
Where is the `/:id` here coming from?

Reply:
Yeah, @BMariscal, i said it in my commit message that, redirecting to the originating page, that is `/feed` isn't working for me, hence the need to place the `/:id` , there, but still needs to be worked on. So i was only able to work on the popup message, but the link still need more logic to work.